src/pages/index.tsx

```diff
@@ -42,94 +41,41 @@ const IndexPage = () => (
     <Highlights
       highlight={[
         {
-          visual: 'hydra',
-          title: 'OAuth 2.0 and OpenID Connect',
-          href: '/hydra',
+          visual: 'opensource',
+          title: 'Powered by Open Source',
+          href: 'https://github.com/ory',
+          openInNewWindow: true,
           content:
-            'Keep threats out and easily verify who is a customer, and customize sign in flows that fit you and your customers.'
+            'You can rely on open source software that is built together in a world-wide community of developers. Peer reviewed, vetted by expert developers - tried and tested.'
         },
         {
-          visual: 'keto',
-          title: 'Authorization',
-          href: '/keto',
+          visual: 'integration',
+          title: 'Easy integration',
+          href: 'https://github.com/ory/sdk',
+          openInNewWindow: true,
           content:
-            'Personalize experiences using customer specific access rules, define roles, and give customers individual access to your service.'
+            'Work with what you’re doing. Easy ways to make tools, software, platforms and coding languages used by your business already work with Ory.'
```

Comment:
The second sentence is quite cumbersome, it took me 5 reads to get it. Maybe change to a different verb than `make .................................................................................................... work`.
```suggestion
            'Work with what you’re doing. Ory enables integrations with tools, software, platforms and coding languages used by your business already.'
```

Reply:
Maybe "integreations"? :joy: